src/lib/client-file-uploader.ts

```diff
@@ -413,10 +413,17 @@ async function getSignedUploadRequestData( {
 	uploadId = undefined,
 	partNumber = undefined,
 }: GetSignedUploadRequestDataArgs ): Promise< PresignedRequest > {
-	const response = await http( '/upload/site-import-presigned-url', {
+	const WPVIP_DEPLOY_TOKEN = process.env.WPVIP_DEPLOY_TOKEN;
+	const reqOptions = {
 		method: 'POST',
 		body: { action, appId, basename, envId, etagResults, partNumber, uploadId },
-	} );
+	} as FetchOptions;
+	if ( WPVIP_DEPLOY_TOKEN ) {
```

Comment:
@rebeccahum @WPprodigy If we don't have a deploy token, maybe we should error out?

Reply:
This part is used flexibly in other pieces, like SQL imports. So can't error here, but we do error out early if the token is missing during the deploy flow.